decohub/apps/hackaton.ts

```diff
@@ -0,0 +1 @@
+export { default } from "../../hackaton/mod.ts";
```

Comment:
what's this hackathon app for?

Reply:
@tlgimenes Sorry, was using our repository as a solution to hackathon, now it's fixed!